src/main/java/Mars.java

```diff
@@ -0,0 +1,205 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class Mars {
+    public static void main(String[] args) {
+        List<Task> arrayList = new ArrayList<>();
+        /* PRE- COMMITS
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+         */
+
+        /* LEVEL 0. Rename, Greet, Exit
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Mars\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________"); */
+        try{
```

Comment:
```suggestion
        try {
```

A space should be inserted here.